config/configload/merge.go

```diff
@@ -13,13 +13,18 @@ import (
 	"github.com/zclconf/go-cty/cty"
 )
 
-func errorUniqueLabels(block *hclsyntax.Block) error {
+const (
+	errMultipleBackends = "Multiple definitions of backend are not allowed."
+	errUniqueLabels     = "All %s blocks must have unique labels."
+)
+
+func createError(msg string, block *hclsyntax.Block) error {
```

Comment:
golang pattern would be `newError` even better `newMergeError`

Reply:
is renamed